src/features/UsageMeter/TierUsageDisplay.tsx

```
@@ -0,0 +1,161 @@
+/**
+ * Tier Usage Display Component
+ * Shows daily Tier 2/Tier 3 usage limits
+ *
+ * Based on PRICING_MASTERPLAN.md.md
+ */
+'use client';
+
+import { Progress, Skeleton, Tooltip, Typography } from 'antd';
+import { createStyles } from 'antd-style';
+import { Infinity, Sparkles, Zap } from 'lucide-react';
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)